standardizeAppFilename.sh

```diff
@@ -47,14 +47,15 @@ for file in "$@"; do
 
     filename=${CFBundleIdentifier_now}_v${CFBundleVersion_now}_${timestamp}.ipa
   else
-    # fix path for gitlab runner
-    echo hostname
-    if [[ $(hostname) = runner-* ]]; then
-      s=$($ANDROID_HOME/build-tools/*/aapt dump badging ${file} | grep package:\ name)
-    else
-      s=$(/usr/local/bin/aapt dump badging ${file} | grep package:\ name)
+    aaptPath=$(find $ANDROID_HOME/build-tools -name aapt -type f | tail -n 1)
```

Comment:
You could also use ` -quit` instead of tail to have find stop after a single match

Reply:
Apparently not on MacOS. I can't find the parameter `-quit` in the man pages on my mac. When testing with 
```
find $ANDROID_HOME/build-tools -name aapt -type f -quit
```
I get zero results.

Reply:
Sorry, forgot to mention that you also have to add `-print` in this case